main.go

```diff
@@ -213,12 +235,15 @@ func imagesSuggestion() []prompt.Suggest {
 
 func main() {
 	dockerClient, _ = docker.NewEnvClient()
-	if _, err := dockerClient.Ping(context.Background()); err != nil {
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
+	defer cancel()
```

Comment:
My connection was broken and the shell was not started. So I've added this line to prevent timeout problems. It can be added or not, it's your choice.